src/crate/qa/tests.py

```diff
@@ -173,11 +173,11 @@ def _unicast_hosts(self, num, transport_port=4300):
             for x in range(num)
         ])
 
-    def _new_cluster(self, version, num_nodes, settings=None, env=None):
+    def _new_cluster(self, version, num_nodes, crate_root=None, settings=None, env=None):
```

Comment:
It looks like `crate_root` is `None` on all call sites. Do we really need this new parameter?